eng/versioning/external_dependencies.txt

```diff
@@ -53,8 +53,10 @@ com.microsoft.azure.functions:azure-functions-java-library;2.2.0
 com.mysql:mysql-connector-j;9.0.0
 com.squareup.okhttp3:mockwebserver;4.12.0
 com.squareup.okhttp3:okhttp;4.12.0
+com.squareup:javapoet;1.13.0
 commons-codec:commons-codec;1.15
 commons-net:commons-net;3.9.0
+io.clientcore.tools:sdk-codegen-tool;1.0.0-beta.1
```

Comment:
I think we can just put this under the same `io.clientcore` groupId - we don't need another sub-group